drivers/eaton-ups-pwnm2-mib.c

```diff
@@ -97,33 +99,35 @@
 #define DEFAULT_SHUTDOWNDELAY	0
 
 static info_lkp_t pw_alarm_ob[] = {
-	{ 1, "OB", NULL, NULL },
-	{ 2, "", NULL, NULL },
-	{ 0, NULL, NULL, NULL }
+	info_lkp_default(1, "OB"),
+	info_lkp_default(2, ""),
+	info_lkp_sentinel
 };
 
 static info_lkp_t pw_alarm_lb[] = {
-	{ 1, "LB", NULL, NULL },
-	{ 2, "", NULL, NULL },
-	{ 0, NULL, NULL, NULL }
+	info_lkp_default(1, "LB"),
+	info_lkp_default(2, ""),
+	info_lkp_sentinel
 };
 
 static info_lkp_t pw_pwr_info[] = {
-	{   1, ""         /* other */, NULL, NULL },
-	{   2, "OFF"       /* none */, NULL, NULL },
-	{   3, "OL"        /* normal */, NULL, NULL },
-	{   4, "BYPASS"    /* bypass */, NULL, NULL },
-	{   5, "OB"        /* battery */, NULL, NULL },
-	{   6, "OL BOOST"  /* booster */, NULL, NULL },
-	{   7, "OL TRIM"   /* reducer */, NULL, NULL },
-	{   8, "OL"        /* parallel capacity */, NULL, NULL },
-	{   9, "OL"        /* parallel redundancy */, NULL, NULL },
-	{  10, "OL"        /* high efficiency */, NULL, NULL },
-	{  11, "BYPASS"    /* maintenanceBypass */, NULL, NULL },
-	{  11, "OL"        /* essMode */, NULL, NULL },
-	
-	{ 0, NULL, NULL, NULL }
-};             
+	info_lkp_default(1, ""),	/* other */
+	info_lkp_default(2, "OFF"),	/* none */
+	info_lkp_default(3, "OL"),	/* normal */
+	info_lkp_default(4, "BYPASS"),	/* bypass */
+	info_lkp_default(5, "OB"),	/* battery */
+	info_lkp_default(6, "OL BOOST"),	/* booster */
+	info_lkp_default(7, "OL TRIM"),	/* reducer */
+	info_lkp_default(8, "OL"),	/* parallel capacity */
+	info_lkp_default(9, "OL"),	/* parallel redundancy */
+	info_lkp_default(10, "OL"),	/* high efficiency */
+
+	info_lkp_default(11, "BYPASS"),	/* maintenanceBypass */
+	info_lkp_default(11, "OL"),	/* essMode */
+		/* FIXME: is "11" correct here or in the line above? */
```

Comment:
@aquette : the two `11` values here, also in original code before the change - which one is right and which is a typo?

Reply:
Offloaded to #2303